### PR TITLE
Refactoring constants to upper case

### PR DESCRIPTION
**Pull request by edgargarciamacrometa**
Also, I added a timeout for the client.on_change() method and changed the .format to f"string"

### CODE REVIEW

File: docs/collections/documents/tutorials/using-realtime-updates.md
Comment (diana-macrometa) [inline docs/collections/documents/tutorials/using-realtime-updates.md:76]: Why are you making the variables uppercase? I thought lowercase was the accepted standard.

Comment (edgargarciamacrometa) [reply docs/collections/documents/tutorials/using-realtime-updates.md:76]: According to [PEP](https://peps.python.org/pep-0008/#constants), constants should be written in uppercase. The support team has been making changes for the pyC8 audit using as a guide the previous link to keep an internal standard. There is another [file](https://github.com/Macrometacorp/docs/blob/master/docs/streams/tutorials/using-realtime-updates.md) within the docs, which has the same content as this. Maybe part of the concern is that we have a #Variables comment in the python file, but they are constants since they do not change their value over time for this example.

Comment (diana-macrometa) [reply docs/collections/documents/tutorials/using-realtime-updates.md:76]: Cool. I didn't realize variables needed to change value in order to be variables.  Learn something new every day. :)

Would they not be variables because the constant would change for different users? Like my email is different than your email, so our variable/constant would have a different value.

**Review by diana-macrometa — APPROVED** [review]

File: docs/collections/documents/tutorials/using-realtime-updates.md
Comment (edgargarciamacrometa) [reply docs/collections/documents/tutorials/using-realtime-updates.md:76]: I understand your point, and I think it can even get more complex. Maybe the word I should use is mutable and immutable objects in python. But in another context, we were also suggesting these changes because of the use of a Linter, in this case, Pylint. It led us to keep almost the same structure for all the files.

Comment (diana-macrometa) [reply docs/collections/documents/tutorials/using-realtime-updates.md:76]: Hey, I like consistency. Thanks for explaining it. :)

Comment (diana-macrometa) [reply docs/collections/documents/tutorials/using-realtime-updates.md:76]: You have an approval, so feel free to merge this once all tests pass.

**Review by jamesm-macrometa — APPROVED** [review]
Looks good!

**Pull request merged**
